app/src/main/java/net/osmtracker/activity/ButtonsPresets.java

```diff
@@ -281,11 +281,7 @@ public void onClick(DialogInterface dialog, int which) {
                                     + Preferences.ICONS_DIR_SUFFIX;
                             File iconDirToDelete = new File(Environment.getExternalStorageDirectory(), rootDir + iconDirName);
 
-                            if(FileSystemUtils.delete(iconDirToDelete, true)){
-                                Toast.makeText(getApplicationContext(), getResources().getString(R.string.buttons_presets_icon_directory_deleted), Toast.LENGTH_SHORT).show();
-                            }else{
-                                Toast.makeText(getApplicationContext(), getResources().getString(R.string.buttons_presets_icon_directory_does_not_exist), Toast.LENGTH_SHORT).show();
-                            }
+                            FileSystemUtils.delete(iconDirToDelete, true);
```

Comment:
why toasts are deleted?

Reply:
There was a problem in showing two Toast consecutively, depending on the phone the tests run on they could appear in different order making it imposible to test consistently  so I found that it would be better to show just one with whether success o error message